rss2email/main.py

```diff
@@ -88,6 +88,9 @@ def run(*args, **kwargs):
     add_parser.add_argument(
         'email', nargs='?',
         help='target email for the new feed')
+    add_parser.add_argument(
+        'from_email', nargs='?',
```

Comment:
This should be `from-email` to be consistent.

Reply:
OK. I'll change it